superbuild/projects_modules/ITK.cmake

```diff
@@ -76,7 +76,9 @@ set(cmake_args
 ## #############################################################################
   
 if (WIN32 AND ${CMAKE_GENERATOR} STREQUAL "Visual Studio 16 2019")
-    ep_GeneratePatchCommand(${ep} ${ep}_PATCH_COMMAND ITK.patch)
```

Comment:
i do not see the ITK_Win patch in the master branch

Reply:
It's called `ITK.patch` in the master branch, and renamed `ITK_Win.patch` in this PR.

Reply:
ah yes indeed sorry i did not see the line changing the name